.github/workflows/push.yml

```diff
@@ -46,11 +46,11 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     - name: Create Release
       if: startsWith(github.ref, 'refs/tags/v')
-      uses: actions/create-release@v1
+      uses: ncipollo/release-action@v1
       with:
-        tag_name: ${{ github.ref }}
-        release_name: ${{ github.ref }}
+        name: ${{ env.VERSION }}
```

Comment:
I don't think the name's necessary
> name: An optional name for the release. If this is omitted the tag will be used.
> tag: An optional tag for the release. If this is omitted the git ref will be used (if it is a tag).

Reply:
Checked again, name isn't required even though earlier testing suggested it was.